frame/staking/src/benchmarking.rs

```diff
@@ -644,7 +644,7 @@ benchmarks! {
 		assert!(value * l.into() + origin_weight <= dest_weight);
 		let unlock_chunk = UnlockChunk::<BalanceOf<T>> {
 			value,
-			era: EraIndex::zero(),
+			unlock_era: EraIndex::zero(),
```

Comment:
I just realized this is an unnecessary breakage for clients..  perhaps we should revert.

Reply:
how so? Its just a name change to struct field, so old type defs should work as well when decoding

Reply:
maybe my definition of breaking was bad: any js code that does `ledger.era` now won't work right?

Reply:
ah yeah good point, unlockChunk.era would break